### PR TITLE
Add description that is missing on PyPi

### PR DESCRIPTION
**Pull request by omi10859**
Check our demo on a test instance of PyPi : https://test.pypi.org/project/spdx-tools

### CODE REVIEW

**Comment by omi10859** [comment]
@pombredanne seems to be working now.

File: setup.py [hunk 1]
```diff
@@ -5,16 +5,21 @@
 
 from setuptools import setup
 import unittest
+from os import path
```
Comment (rtgdk) [inline setup.py]: This import is not needed.

**Comment by omi10859** [comment]
@pombredanne review request

**Comment by omi10859** [comment]
@rtgdk can you please explain what does this commit mean "Merge branch 'master' into description_pypi"

**Pull request merged**